packages/odyssey-react-mui/src/Breadcrumbs.tsx

```diff
@@ -39,6 +39,7 @@ export type BreadcrumbProps = {
   href: string;
   iconName?: "user" | "group";
   onClick?: MouseEventHandler;
+  noninteractive?: boolean;
```

Comment:
This should probably be `hasInteractivity: false` no matter how weird that is, with it defaulted to `true`.

Our API wasn't designed this way, but I don't want to add a negative prop either. In the future, we can default these to `false`.

Reply:
Better yet, we should reconsider adding this prop altogether and look at making `href` optional.